### PR TITLE
Arduino feature implementation

### PR DESCRIPTION
**Pull request by AdrianSch**
Finally we were able to get a green checkstyle run on Jenkins, but the following lines should be checked (pretty ugly): ArduinoImpl.java line 264 - 268 (Martin Burtscher + Me couldn't implement it in another way)

### CODE REVIEW

**Comment by AdrianSch** [comment]
Build #3754 - feature/arduino_firmata_jenkins (01.10.2015 15:55:17)  (https://jenkins.catrob.at/job/Catroid-Build-SourceTests-StaticChecks/3754/) Catroid-Build-SourceTests-StaticChecks were green (of course)

File: catroid/res/values/styles.xml [hunk 1]
```diff
@@ -376,11 +397,6 @@
         <item name="android:background" >@drawable/brick_cyan_3h</item >
     </style >
 
-    <style name="BrickContainer.Phiro.Light" >
```
Comment (thmq) [inline catroid/res/values/styles.xml]: @AdrianSch why did you remove the phrio styles? are they now contained within arduino styles?

File: catroid/src/org/catrobat/catroid/content/Script.java [hunk 2]
```diff
@@ -201,16 +202,14 @@ public int containsBrickOfTypeReturnsFirstIndex(Class<?> type) {
...
-	//					|| (brick instanceof NXTMotorStopBrick) || (brick instanceof NXTPlayToneBrick)) {
-	//				return true;
-	//			}
-	//		}
-	//		return false;
-	//	}
+	public boolean containsBluetoothArduinoBrick() {
+		for (Brick brick : brickList) {
+			if (brick instanceof ArduinoSendDigitalValueBrick) {
+				return true;
```
Comment (thmq) [inline catroid/src/org/catrobat/catroid/content/Script.java]: @AdrianSch please remove the blank lines between the brackets.
